tests/connectors/studio/test_studio.py

```diff
@@ -56,6 +60,24 @@ def studio_client() -> StudioClient:
     return client
 
 
+@fixture
+def examples() -> Sequence[Example[str, str]]:
+    return [
+        Example(input="input_str", expected_output="output_str"),
+        Example(input="input_str2", expected_output="output_str2"),
+    ]
```

Comment:
have you checked if we have this fixture already? otherwise not a big deal to redefine it if it does not fit

Reply:
Yes, but in the "wrong" conftest.